darwin-configuration.nix

```diff
@@ -50,6 +52,9 @@
     postgresql.fastUnsafe = true;
   };
 
+  # Kibana doesn't build on M1, so skip that
+  services.kibana.enable = lib.mkForce (! pkgs.stdenv.isAarch64);
```

Comment:
This should go in the backend. I've been accumulating a nix refactoring branch, so I've included it there: https://github.com/iknow/eikaiwa_content/pull/1838